Model/DutyCalculator.php

```diff
@@ -585,24 +586,18 @@ protected function prepareRequest($freightCharge, $shippingAddress)
     protected function getShippingOrigin()
     {
         $origin = [];
-        $region = $this->_scopeConfig->getValue(
-            self::XML_PATH_ORIGIN_REGION_ID,
-            \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
-            $this->storeManager->getStore()->getId()
-        );
 
+        $region = $this->reachHelper->getShippingOriginState(self::XML_PATH_ORIGIN_REGION_ID,
```

Comment:
would you happen to know why the XML_PATH_ORIGIN_COUNTRY_ID and XML_PATH_ORIGIN_COUNTRY_ID constants are defined in this file rather than with the rest of the paths in the Helper/Data.php file? 

I'm wondering if it would be cleaner to move these paths to the Data.php file the way the rest of the configuration details are so that these calls can go from:

```
$this->reachHelper->getShippingOriginCountry(
            self::XML_PATH_ORIGIN_COUNTRY_ID, $this->storeManager->getStore()->getId());
```

to

`$this->reachHelper->getShippingOriginCountry();`

and line up with the other config getter functions. What do you think?

Reply:
Original developer took some shortcut I think ... could be easily shifted to Data.php. I left the constant/it there as is -  just shifted the getter logic to Data.php for proper handling of that variable with respect to scope (default, website, store). Just changed the minimum that I must need to change/move.

Reply:
For sure I can shift those to data.php and can make it similar to the other getters ... would need another extra round of (env setup and) testing at my end though

Reply:
May be we can do that on that batch of tickets (with a small ticket for this one) when we are pushing dynamic scoping of CC and PayPal?

Reply:
It's certainly not required for this review to be completed. So if you would like to move those constants, awesome! But if not, or you would like to save that work for later that's totally ok too.